cli/src/templates/packages/expo-router/drawer/app/(drawer)/(tabs)/TabTwo.tsx.ejs

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```

Comment:
What are these files for?

Reply:
Minimal screens for bottom tabs

Reply:
is there a reason that you chose to create separate files for these components vs just declaring them inline?

Reply:
Usually React Navigation leaves warning for inline screens, so it is probably habit.
But also easier for people to copy paste file of screen and start changing.

Most of these changes in PR are bare minimum for you guys to review my approach to this CLI.